sceptre/plan/plan.py

```diff
@@ -376,3 +376,13 @@ def diff(self, *args) -> Dict[Stack, StackDiff]:
         """
         self.resolve(command=self.diff.__name__)
         return self._execute(*args)
+
+    def drift_show(self, *args):
```

Comment:
Same comment about the name here; I think it really ought to be called something like "detect_resource_drift"